integration-tests/actions/actions.go

```diff
@@ -200,7 +200,7 @@ func TeardownSuite(
 	} else {
 		log.Info().Msg("Successfully returned funds from chainlink nodes to default network wallets")
 	}
-	// nolint
+	//nolint:all
```

Comment:
What is this one suppressing? :thinking:

Reply:
Good question... looks a lot like the one I was getting on the last PR.  I tried removing the `//nolint:all` directive, and I didn't see any new errors from `golangci-lint`.  That gave me the idea to remove all the others and see which ones are actually doing anything.  Apparently only a handful were... I've re-enabled the ones which were working, and made them more specific to only apply to one kind of check on a particular line, instead of a whole block.

For making comparisons, I added `--max-issues-per-linter 10000 --max-same-issues 10000` to the commandline, because otherwise I think it only reports a subset of the actual errors.  We still have lots of outstanding issues, but I made sure at least no new ones show up with these changes.

Reply:
I think we accumulated a bunch during a period of over-tuned linting that we've since dialed back.

You can use `0` for unlimited too. See `make golangci-lint`: https://github.com/smartcontractkit/chainlink/blob/5dbded53edb9ca2f1549fe133c4918dfc5cc94d7/GNUmakefile#L151